src/test/java/io/strimzi/test/container/StrimziKafkaContainerIT.java

```diff
@@ -72,7 +72,7 @@ void testStartContainerWithEmptyConfiguration(final String imageName) {
             systemUnderTest.start();
 
             assertThat(systemUnderTest.getBootstrapServers(), is("PLAINTEXT://"
-                    + systemUnderTest.getContainerIpAddress() + ":" + systemUnderTest.getMappedPort(9092)));
+                    + systemUnderTest.getHost() + ":" + systemUnderTest.getMappedPort(9092)));
```

Comment:
`getContainerIpAddress()` is deprecated, so switching to `getHost()` which is the preferred method now.